llvm/lib/Transforms/Yk/ControlPoint.cpp

```diff
@@ -88,6 +88,11 @@
 #define DEBUG_TYPE "yk-control-point"
 #define JIT_STATE_PREFIX "jit-state: "
 
+// These constants mirror `ykrt::mt::JITACTION_*`.
+#define JITACTION_NOP 1
```

Comment:
This might be subtly dangerous: these are strings not integers, so if someone uses them in a not-compatible-with-Rust's-`usize` context they could get weird results. Looking at the rest of the commit, we do the right thing, so I think we can just add a comment here along the lines of "These must only be used in an integer context which is compatible with Rust's `usize` as that is the type of `ykrt::mt::JITACTION_*`."

Reply:
I don't follow what you mean with strings.

Do you mean, it's a macro, so it's textually substituted?

Reply:
> Do you mean, it's a macro, so it's textually substituted?

Yes.

Reply:
I know what you are saying, but I think that just comes with the territory.

You don't see people writing a comment like that every time they define an integer constant using a macro.

Regardless, we can probably fix this with C++ isms. Leave it with me.

Reply:
> You don't see people writing a comment like that every time they define an integer constant using a macro.

That's because they're sloppy ;)

I don't see a big problem though, and can we not use my suggestion of just adding a comment "These must only be used in an integer context which is compatible with Rust's usize as that is the type of ykrt::mt::JITACTION_*." ?

Reply:
We've decided not to do this.

Reply:
Fixed in 571fd63e2df7 contains a change that converts those macro defs to typed constants (same type as on the Rust side). I think that's probably better?